Classes/New Issue/GithubClient+Template.swift

```diff
@@ -0,0 +1,130 @@
+//
+//  GithubClient+Template.swift
+//  Freetime
+//
+//  Created by Ehud Adler on 11/11/18.
+//  Copyright © 2018 Ryan Nystrom. All rights reserved.
+//
+
+import Foundation
+import GitHubAPI
+import GitHubSession
+import Squawk
+
+extension GithubClient {
+
+    private func fetchTemplateFile(
+        repo: RepositoryDetails,
+        filename: String,
+        completion: @escaping (Result<String>) -> Void
+        ) {
+
+        self.fetchFile(
+            owner: repo.owner,
+            repo: repo.name,
+            branch: repo.defaultBranch,
+            path: "\(Constants.Filepaths.githubIssue)/\(filename)") { (result) in
```

Comment:
If this gets rebased onto `master` I believe @rnystrom has created a URLBuilder to make this a bit nicer

Reply:
I think URLBuilder makes a full url (Https/http:// ... ). I'll check it out

Reply:
Ya URLBuilder seems to require a scheme. I also don't see why URLBuilder is better than a constant. Mind if I keep this for now?

Reply:
Not at all, hadn't looked into the URLBuilder stuff and wondered if it would've worked! All good for now